share/completions/nmcli.fish

```diff
@@ -1,3 +1,42 @@
+function __fish_nmcli_list_field_values
+    set -l command
+    set -l description
+
+    for i in $argv
+        switch $i
+            case id
+                set command "nmcli -g NAME connection show"
+                set description "Connection name"
+            case uuid
+                set command "nmcli -g UUID connection show"
+                set description "Connection UUID"
+            case path
+                set command "nmcli -g DBUS-PATH connection show"
+                set description "D-Bus static connection path"
+            case apath
+                set command "nmcli -g ACTIVE-PATH connection show --active"
+                set description "D-Bus active connection path"
+            case ifname
+                set command "nmcli -g DEVICE device status"
+                set description "Interface name"
+            case ssid
+                set command "nmcli -g SSID device wifi list"
+                set description "SSID"
+            case bssid
+                set command "nmcli -g BSSID device wifi list"
+                set description "BSSID"
+            case '*'
+                return 1
+        end
+
+        if $command | string length -q
+            printf '%s\n' ($command | string trim | string replace -rf '^(.+)$' '$1\t'$description'')
```

Comment:
1. This will run it like `"nmcli -g BSSID device wifi list"`, which will look for an executable named "nmcli -g BSSID device wifi list", with the spaces. Fish does not do word-splitting on variables.

2. This will run $command _twice_ - it'll check if the command produces output and then will run it again to handle the output.

3. This will split the command's output on newlines and then add the newlines back again (the printf is unnecessary).

Use `set command nmcli -g BSSID device wifi list`, and then just always run

```fish
$command | string trim | string replace -rf '^(.+)$' '$1\t'$description''
```

If $command is empty you already did `return 1` before.